docs/examples/DataSet/Measuring X as a function of time.ipynb

```diff
@@ -0,0 +1,179 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Measuring X as a function of time\n",
+    "\n",
+    "Sometimes we'd like to measure something as a function of elapsed wall clock time. This notebook provides a simple example of doing that using the `ElapsedTimeParameter`."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import os\n",
+    "\n",
+    "import numpy as np\n",
+    "\n",
+    "from qcodes.utils.time import ElapsedTimeParameter\n",
+    "from qcodes.instrument.parameter import Parameter\n",
+    "from qcodes.dataset.measurements import Measurement\n",
+    "from qcodes.dataset.sqlite.database import initialise_or_create_database_at\n",
+    "from qcodes.dataset.experiment_container import new_experiment\n",
+    "from qcodes.dataset.plotting import plot_dataset"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Prepatory footwork: setup database and experiment"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "tutorial#no_sample#5@C:\\Users\\wihpniel\\src\\qcodes\\docs\\examples\\DataSet\\x_as_a_function_of_time.db\n",
+       "--------------------------------------------------------------------------------------------------"
+      ]
+     },
+     "execution_count": 2,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "initialise_or_create_database_at(os.path.join(os.getcwd(), 'x_as_a_function_of_time.db'))\n",
+    "new_experiment('tutorial', 'no_sample')"
```

Comment:
I changed most (all) of the other example notebooks around to use load_or_create_experiment and explicitly pass that to the measurement. I think this is the best solution until we have an explicit experiment switcher

Reply:
Absolutely.